docs/tutorials/eda/eda-auto-analyze-interaction.ipynb

```diff
@@ -44,7 +44,7 @@
   {
```

Comment:
<p>Typo:</p><p> </p><blockquote><code>Fare</code>&nbsp;and&nbsp;<code>Age</code>: **bot** have a non-linear relationship with the outcome;</blockquote><blockquote><code>Fare</code>&nbsp;**and** has two modes&nbsp;</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/autogluon/autogluon/pull/3071/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
fixed